extensions/mongoProcessor/MongoProcessorMetrics.js

```diff
@@ -0,0 +1,46 @@
+const { ZenkoMetrics } = require('arsenal').metrics;
+
+const config = require('../../lib/Config');
+const { promMetricNames } = require('./constants');
+
+const ingestionProcessedElapsedSeconds = ZenkoMetrics.createHistogram({
+    name: promMetricNames.ingestionProcessedElapsedSeconds,
+    help: 'Duration of the process ingestion jobs in seconds',
+    buckets: promMetricNames.ingestionBucketTimes,
+    labelNames: ['bucketName', 'locationName', 'locationType', 'status'],
```

Comment:
Cardinality of bucket names and location names is defined by the user, not by the system. Hence, should this be pushed into Prometheus time series?

Reply:
In this specific case, OOB is setup per location. One location maps to one bucket, so the cardinality is static. This should be ok, right?

Reply:
There are 2 different issues that can contribute to issues (prometheus anti-patterns):
- some of the labels have unbounded cardinality (`bucketName` and `locationType` even if they are 1:1 between themselves)
- there are 4 different labels

These 2 issues combined can create a lot of distinct time series that can hurt storage efficiency and query performance.

Reply:
bucket name, location name and location type labels are not required for our OOB Ingestion processor dashboard's panel. 
I decided to remove them for now.